src/main/fc/fc_tasks.c

```diff
@@ -139,6 +143,14 @@ static void taskUpdateRxMain(timeUs_t currentTimeUs)
 
     isRXDataNew = true;
 
+#ifdef USB_CDC_HID
+    int8_t report[8];
```

Comment:
this should be probably disabled when ARMed

Reply:
We can do it indeed:)